zh_CN/readme.md

```diff
@@ -0,0 +1,955 @@
+
+断言`value`是否和`expected`相等。
+
+### `.not(value, expected, [message])`
+
+断言`value`是否和`expected`不等。
+
+### `.same(value, expected, [message])`
+
+断言`value`是否和`expected`深度相等。
+
+### `.notSame(value, expected, [message])`
+
+断言`value`是否和`expected`深度不等。
+
+### `.throws(function|promise, [error, [message]])`
+
+断言`function`抛出一个异常，或者`promise` reject 一个错误。
+
+`error`可以是一个构造器，正则，错误信息或者验证函数。
+
+返回由`function`抛出的异常或`promise`的拒绝原因。
+
+### `.notThrows(function|promise, [message])`
+
+断言`function`没有抛出一个异常，或者`promise` resolve。
+
+### `.regex(contents, regex, [message])`
+
+断言`contents`匹配`regex`。
+
+### `.ifError(error, [message])`
+
+断言`error`是假值。
+
+### 跳过断言
+
+通过`skip`修饰符可以跳过任何断言，跳过的断言仍然会被计数，所以不需要去改变你的断言计划数量。
+
+```js
+test(t => {
+    t.plan(2);
+    t.skip.is(foo(), 5); // 当跳过时不需要改变你的计划数
+    t.is(1, 1);
+});
+```
+
+### 强化断言信息
+
+AVA 自带内置的[`power-assert`](https://github.com/power-assert-js/power-assert)，给你更多的描述性断言信息，它阅读你的测试代码并从中试图推断出更多信息。
+
+我们来举个例子，使用Node 的标准[`断言`库](https://nodejs.org/api/assert.html)。
+
+```js
+const a = /foo/;
+const b = 'bar';
+const c = 'baz';
+require('assert').ok(a.test(b) || b === c);
+```
+
+如果你把代码粘贴到Node REPL 中会返回：
+
+```
+AssertionError: false == true
+```
+
+而在AVA 中，这个测试：
+
+```js
+test(t => {
+    const a = /foo/;
+    const b = 'bar';
+    const c = 'baz';
+    t.ok(a.test(b) || b === c);
+});
+```
+
+将会输出：
+
+```
+t.ok(a.test(b) || b === c)
+       |    |     |     |
+       |    "bar" "bar" "baz"
+       false
+```
+
+## 隔离进程
+
+每个测试文件都会在一个独立的Node 进程中运行，这样让你可以在一个测试文件中改变全局状态或将其覆盖一个内置的全局状态，而不会影响其他测试文件。这样更加有效地利用现代的多核处理器，让多个测试可以并发地执行。
+
+## 小贴士
+
+### 临时文件
+
+并发运行测试面临着许多挑战，文件的IO 操作就是其中一个。
+
+一般来讲，串行测试在当前文件夹中创建临时文件夹，然后会在测试结束的时候清理它们。但这种做法在并发测试中不起作用，因为测试会一起操作这个文件夹而导致冲突。正确的做法是为每个测试创建一个新的临时文件夹，使用[`tempfile`](https://github.com/sindresorhus/tempfile)和[`temp-write`](https://github.com/sindresorhus/temp-write) 模块可能会有帮助。
+
+### 调试
+
+AVA 默认情况下是并发执行测试，这样在调试信息时并不是最理想的，你可以通过设置`--serial`选项来让测试串行执行。
+
+```console
+$ ava --serial
+```
+
+### 代码覆盖率
+
+你不能使用[`istanbul`](https://github.com/gotwarlost/istanbul)来做代码覆盖率因为AVA[处理过这些测试文件](#隔离进程)，你可以使用[`nyc`](https://github.com/bcoe/nyc)来代替，你可以把它看做是支持子进程的`istanbul`。 
+
+从版本`5.0.0`开始，在报告覆盖率时为你的代码使用原生映射，而不是转换后的代码。确保你测试的代码包括了一个内联的原生映射或者引用了一个原生映射文件。如果你使用了`babel-register`，你可以在你的Babel 配置中将`sourceMaps`设置为`inline`。
+
+## FAQ
+
+### 为什么不用`mocha`，`tape`，`tap`？
+
+Mocha 要求你使用隐式全局变量比如`describe`和`it`作为其默认接口（这是大部分人使用的）。这样做不是很好，并且串行执行测试没有进程隔离，使得测试十分缓慢。
+
+Tape 和tap 是非常好的。AVA 在它们的语法中得到大量启发，但它们也是串行执行测试，它们的默认[TAP](https://testanything.org) 输出不是非常友好，因此你总是需要使用额外的tap 报告。
+
+与它们不同的是，AVA 可以并发执行测试，为每个测试文件提供独立进程，它的默认报告简单明了，并且AVA 也支持通过CLI 标志来输出一个TAP 报告。
+
+### 我如何使用自定义的报告？
+
+AVA 支持TAP 格式，所以它兼容任何[TAP 报告](https://github.com/sindresorhus/awesome-tap#reporters)，使用[`--tap`标志](#可选的tap 输出)来启用TAP 输出。
+
+### 项目名字要怎么写才是正确的？如何发音？
+
+AVA，不是Ava，也不是ava，发音[`/ˈeɪvə/` ay-və](https://github.com/sindresorhus/ava/blob/master/media/pronunciation.m4a?raw=true)。
+
+### 项目背景图片是什么意思？
+
+它是[处女座星系](https://simple.wikipedia.org/wiki/Andromeda_galaxy)。
```

Comment:
Replace:
`https://simple.wikipedia.org/wiki/Andromeda_galaxy`
by this chinese link is better:
`https://zh.wikipedia.org/wiki/%E4%BB%99%E5%A5%B3%E5%BA%A7%E6%98%9F%E7%B3%BB`